okhttp-testing-support/build.gradle

```diff
@@ -2,8 +2,13 @@ dependencies {
   api project(':okhttp')
   api deps.junit
   api deps.assertj
-  api deps.bouncycastle
-  api deps.bouncycastletls
+  api deps.bouncycastledebug
+  implementation deps.bouncycastlepkix, {
+    exclude module:"bcprov-jdk15on"
+  }
+  implementation deps.bouncycastletls, {
+    exclude module:"bcprov-jdk15on"
```

Comment:
could you add a comment to explain this? Dependency excludes are always hiding problems as far as I’m concerned~

Reply:
It's about running with the debug version which does some logging.  But otherwise you get randomly logging or not when debugging.

Reply:
oh, that’s totally interesting. What signifies the debug version?

Reply:
'bouncycastle': "org.bouncycastle:bcprov-jdk15on:${versions.bouncycastle}",
      'bouncycastledebug': "org.bouncycastle:bcprov-debug-jdk15on:${versions.bouncycastle}",

These two are the same with additional debug.  It's minimal, but given how bad every other SSL library is at debugability, it's something...

Reply:
discussed here https://www.bouncycastle.org/latest_releases.html